docs/book/src/user/quick-start.md

```diff
@@ -93,6 +93,45 @@ a target [management cluster] on the selected [infrastructure provider].
    Then follow the instruction for your kind version using  `kind create cluster --config kind-cluster-with-extramounts.yaml`
    to create the management cluster using the above file.
 
+   {{#/tab }}
+   {{#tab KubeVirt}}
+
+   #### Create the Kind Cluster
+   To allow using LoadBalancer, we can't use the kind's default CNI (kindnet), but we'll need to install another CNI, like Calico. In order to do that, we'll need first to initiate the kind cluster with two modifications:
+   1. Disable the default CNI
+   2. Add the docker credentials to the cluster, to avoid the docker hub pull rate limit.
+   
+   Create a configuration file for kind. Please notice the docker config file path, and adjust it to your local setting:
+   ```bash
+   cat <<EOF > kind-config.yaml
+   kind: Cluster
+   apiVersion: kind.x-k8s.io/v1alpha4
+   networking:
+   # the default CNI will not be installed
+     disableDefaultCNI: true
+   nodes:
+   - role: control-plane
+     extraMounts:
+      - containerPath: /var/lib/kubelet/config.json
+        hostPath: <YOUR DOCKER CONFIG FILE PATH>
+   EOF
+   ```
+   Now, create the kind cluster with the configuration file:
+   ```bash
+   kind create cluster --config=kind-config.yaml
+   ```
+   Test to ensure the local kind cluster is ready:
+   ```bash
+   kubectl cluster-info
+   ```
+
+   #### Install the Calico CNI
+   Download the Calico manifest and use it to create the required resources, as described [here](https://projectcalico.docs.tigera.io/getting-started/kubernetes/self-managed-onprem/onpremises#install-calico); e.g.:
+   ```bash
+   curl https://raw.githubusercontent.com/projectcalico/calico/v3.24.3/manifests/calico.yaml -O
```

Comment:
Do we want to pin Calico to a specific version? It will require frequent updates to the book in order to not get outdated which might make this doc harder to maintain

Reply:
In the "others..." tab it has been proven to be sustainable as we usually bump calico as well when we bump Kubernetes.

I don't know how sustainable it is for kubevirt (as we wouldn't bump that when we bump for others as we don't have the time to test it).

We pin in the "others..." tab to ensure we have something which has been tested manually at least once. We couldn't guarantee this with some sort of "latest".

Reply:
Thanks for your comment @CecileRobertMichon. I was wandering about it when I wrote it. But this is not only a matter of versions, because new calico version my also bring relevant API changes. So I prefer to pin it for now, and monitor it. Any change will require some manual testing (i.e. go over the whole guide again and again to make sure everything is still working).

Reply:
That's fair 👍 SGTM